app/Jobs/NoAttendanceJediPush.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+// phpcs:disable SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingReturnTypeHint
+
+namespace App\Jobs;
+
+use App\Team;
+use App\User;
+use Illuminate\Bus\Queueable;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+
+class NoAttendanceJediPush implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
+
+    /**
+     * Execute the job.
+     *
+     * @return void
+     */
+    public function handle(): void
+    {
+        $users = User::active()->whereDoesntHave('attendance', static function (Builder $query) {
+            $query->where('attendable_type', Team::class)->where('created_at', '>', now()->startOfDay()->subDays(28));
```

Comment:
I would like to avoid syncing a similar group of users to SUMS every day. We could either filter out any users that would have been sent in the previous day's job here, or I can bump up the time limit on the debounce cache to 48 hours.

Reply:
I'll filter by attendance created before 28 days ago and after 29 days ago, that should eliminate the duplication since it'll only match people for one day.